### PR TITLE
Display Corda node legal name in JVisualVM Console for MockNode testing.

### PR DESCRIPTION
**Pull request by josecoll**
Simple but effective and very useful!

### CODE REVIEW

**Comment by mikehearn** [comment]
Ah, great. Might as well use a space instead of a dash so it looks entirely human friendly: it's not machine parsed and legal name can have spaces in it anyway, so it doesn't have to look like an identifier. Could be "Corda node for ${legalName}" for instance.

**Comment by josecoll** [comment]
I'll change the dash for a space - but want to keep it short as you often resize the left hand panel to view the stats in the main window.

File: node/src/main/kotlin/net/corda/node/driver/Driver.kt [hunk 1]
```diff
@@ -566,7 +566,7 @@ class DriverDSL(
 
                 val systemProperties = mapOf(
                         "name" to nodeConf.myLegalName,
-                        "visualvm.display.name" to "Corda"
+                        "visualvm.display.name" to "Corda ${nodeConf.myLegalName}"
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/driver/Driver.kt]: Does `Bank A Corda` sound better than `Corda Bank A`?

Comment (josecoll) [inline node/src/main/kotlin/net/corda/node/driver/Driver.kt]: Visually it looks much nicer having all corda nodes sorted from left to right.

**Comment by josecoll** [comment]
Ironically JVisualVM refused to parse an empty space, so the dash is back in!!!

screen shot 2017-03-28 at 17 38 03

**Comment by mikehearn** [comment]
Legal names can have spaces in them so if VisualVM really can't handle that (super weak) then we'll have to do a whole string replace of ' ' to '-'

**Pull request merged**